python/ray/train/tests/test_mosaic_trainer.py

```diff
@@ -0,0 +1,146 @@
+import pytest
+
+# torch libraries
+import torch
+import torch.utils.data
+
+# torchvision libraries
+import torchvision
+from torchvision import transforms, datasets
+
+# Import Ray libraries
+import ray
+from ray.air.config import ScalingConfig
+import ray.train as train
+from ray.air import session
+
+# import ray-mosaic integration libraries
+from ray.train.mosaic import MosaicTrainer
+
+# import composer training libraries
+from torchmetrics.classification.accuracy import Accuracy
+from composer.core.evaluator import Evaluator
+from composer.models.tasks import ComposerClassifier
+import composer.optim
+from composer.loggers import InMemoryLogger
+from composer.algorithms import LabelSmoothing
```

Comment:
our import convention in order is 

python native imports 
third party imports
ray imports

Reply:
Reformatted